native/acter/src/api/calendar_events.rs

```diff
@@ -237,6 +229,26 @@ impl CalendarEvent {
             })
             .await?
     }
+
+    pub async fn my_rsvp_status_str(&self) -> Result<String> {
+        let client = self.client.clone();
+        let event_id = self.inner.event_id().to_owned();
+        let my_id = self.client.user_id().context("User not found")?;
+
+        RUNTIME
+            .spawn(async move {
+                let manager =
+                    models::RsvpManager::from_store_and_event_id(client.store(), &event_id).await;
+                // read last rsvp
+                for entry in manager.rsvp_entries().await?.into_iter().rev() {
```

Comment:
maybe `rsvp_entries` should actually be a hashmap `userid -> latest status` and then we just overwrite the latest value? Not necessarily something to be changed today though...

Reply:
Fixed by https://github.com/acterglobal/a3/pull/979/commits/65e51eac4581093951a14e1ff44c21eba7004ce9